crawler/config.py

```diff
@@ -9,6 +9,16 @@ class RSSConfig:
     links = {
         "The Verge": "https://www.theverge.com/rss/index.xml",
         "Engadget": "https://www.engadget.com/rss.xml",
-        # Feedburner based XML Feed, need to be careful
+        "LifeHacker": "https://lifehacker.com/rss",
+        "Wired": "https://www.wired.com/feed/rss",
+        # TODO(@ethkim): Fix the XML feed consumption to prevent breaking from browser-friendly style codes
         "TechCrunch": "http://feeds.feedburner.com/TechCrunch/"
+    }
+
+    # Positional ordered: name, publish, content, author
```

Comment:
Are these items the name of sections for each content provider? Then we could say in the comment:

"Each item in the following list contains the name of sections (for each content provider), 
for which we extract content."

Reply:
Ok! I'll fix it. Thank you for advice.